test/job/start.js

```diff
@@ -32,59 +31,73 @@ test('starts/stops the job queue', t => {
     agenda.processEvery('1 second');
 
     await startAgenda(agenda);
-    t.true(agenda._processInterval);
+    t.truthy(agenda._processInterval);
   });
 });
 
-test.skip('does not run disabled jobs', async t => {
+test('does not run disabled jobs', async t => {
   const {agenda} = t.context;
 
-  await stopAgenda(agenda);
-  await clearJobs(agenda);
-
   agenda.define('disabledJob', (job, done) => {
     t.fail();
     done();
   });
 
   const job = agenda.create('disabledJob').disable().schedule('now');
 
-  const {lastRunAt} = promisify(job.save)().then(job => job.attrs);
-
-  await startAgenda(agenda);
+  await new Promise(resolve => {
+    job.save((err, job) => {
+      t.ifError(err);
+      resolve(job);
+    });
+  });
 
-  t.true(lastRunAt === undefined);
+  await new Promise(resolve => {
```

Comment:
I'm not sure I get the reason behind this since this more or less skips the actual test since it should be checking if `lastRunAt` was still undefined.

Reply:
It looked like that check wasn't helpful. I can add it back in if that was incorrect.

My thoughts were that the job should just not run and the t.fail() that is inside defined job accomplishes that.